server/routes/app.controller.js

```diff
@@ -0,0 +1,35 @@
+const httpStatus = require("http-status");
+const Profile = require("../models/profiles.model");
+const moment = require("moment-timezone");
+const APIError = require("../APIError");
+
+/**
+ * Returns success if able to create new user
+ * @public
+ */
+exports.register = async (req, res, next) => {
```

Comment:
why not es6 syntax here?

Reply:
I'm assuming you're talking about named exports vs export default?  Because of sporadic issues with Node and how it handles imports (it doesn't really, which is why all the requires), I've kept to the standard Node module require/export syntax